### DISCUSSION TITLE
Can I open any web page URL and then execute javascript on that page?

### DISCUSSION THREAD

**Discussion by jayenashar** [ionic-team/capacitor#5624] [Q&A] [answered]
As an example, would I be able to open this github discussion and then write some javascript to change all the links (`href` attribute of `a` elements)?

**Comment by w-mazed** [comment, marked as answer]
Hi @jayenashar,

With the **In App Browser** module you can add CSS and also execute Js scripts on any page (but keep in mind that it doesn't affect the server side data)
[https://ionicframework.com/docs/native/in-app-browser](https://ionicframework.com/docs/native/in-app-browser)

You can also check the link bellow to find more details
[https://github.com/apache/cordova-plugin-inappbrowser](https://github.com/apache/cordova-plugin-inappbrowser)

Best regards.
